### ISSUE TITLE
AuthzListViewController remove static accesses to data from the view controller

### ISSUE THREAD

**Issue by albertopeam** [cosmostation/cosmostation-ios#237]
`ViewDidLoad()` makes some access to data 
```swift
override func viewDidLoad() {
        super.viewDidLoad()
        self.account = BaseData.instance.selectAccountById(id: BaseData.instance.getRecentAccountId())
        self.chainType = ChainFactory.getChainType(account!.account_base_chain)
        self.chainConfig = ChainFactory.getChainConfig(chainType)
}
```
it would be nice to move to the reducer/inject in the state and save lines in the ViewController, this way the ViewController won't have logic that access directly to `BaseData`

**Comment by albertopeam** [comment]
I will take my time to do it properly :)
I hope this way we can introduce tests and cleanup code repetition in the future

**Comment by albertopeam** [comment]
close as it was merged in #241
